src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsInterlocDecisionDocument.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Value
+@Builder(toBuilder = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SscsInterlocDecisionDocument {
```

Comment:
The 3 document class definitions all look the same.  Why not just have one SscsDocument class?

Reply:
We are using implicit parser which looks for name SscsInterlocDecisionDocument/SscsInterlocDirectionDocument and we don't have much fields in each class